code/datums/mutantraces.dm

```diff
@@ -290,6 +293,14 @@ ABSTRACT_TYPE(/datum/mutantrace)
 				src.blood_color_changed = TRUE
 
 		src.AH = M.bioHolder?.mobAppearance // i mean its called appearance holder for a reason
+		src.hair_custom_style = list()
+		src.hair_custom_style["hair_icon"] = src.AH?.customization_first.icon
+		src.hair_custom_style["hair_id"] = src.AH?.customization_first.id
+		src.hair_custom_style["beard_icon"] = src.AH?.customization_second.icon
+		src.hair_custom_style["beard_id"] = src.AH?.customization_second.id
+		src.hair_custom_style["detail_icon"] = src.AH?.customization_third.icon
+		src.hair_custom_style["detail_id"] = src.AH?.customization_third.id
```

Comment:
Why does this need to be duplicated here?

Reply:
In human/proc/Equip_Job_Slots(), it sets the customization styles to new /datum/customization_style/none, so need to save the icons and ids. Tried messing around with that in a few places, figured here would be most appropriate

Reply:
Oh there's actually a var for this I think: `AH_we_spawned_with`